CHANGELOG.md

```diff
@@ -23,7 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - oesign sign now allows option -o/--output-file, to specify location to write signature of enclave image.
 
 ### Changed
-- oe_get_attestation_certificate_with_evidence() has been deprecated because it has been deemed insufficient for security. Use the new, experimental oe_get_attestation_certificate_with_evidence_v2() instead to generate a self-signed certificate for use in the TLS handshaking process.
+- `oe_get_attestation_certificate_with_evidence()` has been deprecated because it has been deemed insufficient for security. Use the new, experimental `oe_get_attestation_certificate_with_evidence_v2()` instead to generate a self-signed certificate for use in the TLS handshaking process. Likewise, use the new `oe_verify_attestation_certificate_with_evidence_v2()` instead of the deprecated `oe_verify_attestation_certificate_with_evidence()` to verify a self-signed certificate.
```

Comment:
You may want to rebase against the master. These logs here are going for v0.15, and I think they now should be under unreleased.

Reply:
Good point, thanks!